packages/contracts/docs/developer-portal/04-framework-lifecycle/03-osx-components/02-upgradeable-contract.md

```diff
@@ -0,0 +1,62 @@
+# UUPS Upgradeable Contract Change
+
+Applies to
+
+- `DAO`
+- `PluginRepo`
+
+```mermaid
+flowchart TD
+    processStart("UUPS upgradeable contract")
+    processStart ==> implementation
+    subgraph implementation[Implementation]
+        affectsStorageGaps{"affects sto- \n rage  layout?"}
+        adaptStorageGaps[["adapt storage gaps"]]
+
+        affectsSubgraph{"affects subgraph?"}
```

Comment:
Is this the correct place for this ?

It might be: 
- affects storage && affects subgraph
- or no affects storage && affects subgraph (right ? or i might be missing somthing)

Reply:
If u agree, also re-visit **non-upgradeable-contract.md**

Reply:
> It might be:
> - affects storage && affects subgraph
> - or no affects storage && affects subgraph (right ? or i might be missing somthing)

Uh, right. Fixed in a8abba6de2f8399bdcf4641acc586cd4a7ff72a1

> Do u think that affectsSubgraph should be it's own sub-process?

Yes. For now, processes are centered around the contracts, but we can extend them to other software components.

Reply:
`Yes. For now, processes are centered around the contracts, but we can extend them to other software components.`

I meant like a sub process in the /04-sub-process folder, so we don't need to repeat...etc

but if you feel it will extend the scope of this PR, feel free to create follow up tasks

Reply:
Created https://aragonassociation.atlassian.net/browse/OS-790